_overviews/scala3-macros/tutorial/reflection.md

```diff
@@ -31,11 +41,126 @@ As it is a subtype of `Tree` we can also look into the `TreeMethods` to find mor
 Each type is also a module with some _static-ish_ methods, for example in the [`TypeReprModule`](https://dotty.epfl.ch/api/scala/quoted/Quotes$reflectModule$TypeReprModule.html) we can find the method `TypeRepr.of[T]` which will create an instance of `Type` containing `T`.
 
 
-## Relation with expressions
-<!-- Term vs Expr -->
-<!-- Safty -->
-*Coming soon*
+## Relation with Expr/Type
+
+Expressions `Expr[T]` can be seen as wrapper around a `Term` that knows statically the type of the term.
+Bellow we use the `asTerm` extension method to transform the expression into a term.
+This extension method is only available after importing `quotes.reflect.asTerm`.
+Then we use `asExprOf[Int]` to transform the term back into `Expr[Int]`.
+This operation will fail if the term does not have provided type or if the term is not a valued expression.
```

Comment:
Maybe remind users what a `valued` expression is?

Reply:
It was a typo. It should be `valid expression`.

Reply:
I added an example

Reply:
Great, thanks!